### DISCUSSION TITLE
Why do floating windows appear? How to disable it?

### DISCUSSION THREAD

**Discussion by Blushyes** [tauri-apps/tauri#10748] [Q&A] [answered]
When I'm using Tauri and working with the <input> tag, a floating window similar to the one in the image below appears while typing. How can I close or disable this floating window? 🤔🚫

![QQ_1724406018923](https://github.com/user-attachments/assets/cd63d4a4-388e-46b5-9a72-b14ccfad22cb)

**Comment by FabianLars** [comment, marked as answer]
Closing as duplicate of https://github.com/tauri-apps/tauri/discussions/7635 (please try the suggestion solution from that discussion)
